static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner-utilities.js

```diff
@@ -0,0 +1,69 @@
+define([
```

Comment:
is `membership-engagement-banner-utilities.js` the right name for this ? maybe `membership-engagement-banner-parameters.js` ?